### ISSUE TITLE
Urgent!! Invalid Loss

### ISSUE THREAD

**Issue by jc-barreto** [pierluigiferrari/ssd_keras#384]
Tensorflow ==1.15
Keras = 2.2.4

Running on Google collab. I don't have GPU.

I'm trying to run the notebook [SSD300_training](https://github.com/pierluigiferrari/ssd_keras/blob/master/ssd300_training.ipynbl)
but I'm getting an error of invalid loss. The error is:

````
Epoch 1/10

Epoch 00001: LearningRateScheduler setting learning rate to 0.001.
   6/1000 [..............................] - ETA: 9:26:20 - loss: nan    Batch 5: Invalid loss, terminating training
````

I'm running with adam optimizer. The parameters are:

adam = Adam(lr=0.001, beta_1=0.9, beta_2=0.999, epsilon=1e-08, decay=0.0)
batch_size = 8
initial_epoch   = 0
final_epoch     = 10
steps_per_epoch = 1000

Could someone help me? I really need to run this code
I have tried to chance to sgd optimizer but gives the same error...

**Comment by ManishKakarla** [comment]
did you solved the problem?

**Comment by jc-barreto** [comment]
No... I give up :/

**Comment by ManishKakarla** [comment]
I got a fix for it .I increased the batch size and tried to run with few other changes in the dependency now I am able to train .

**Comment by iagupov** [comment]
@ManishKakarla could you please share waht did change apart from a batch size?
thanks
